api/routes.go

```diff
@@ -5,28 +5,22 @@ import (
 	"net/http"
 
 	"github.com/julienschmidt/httprouter"
-	"gitlab.com/NebulousLabs/errors"
 )
 
 // buildHTTPRoutes registers all HTTP routes and their handlers.
 func (api *API) buildHTTPRoutes() {
-	// TODO Rate-limit these from the start to prevent brute force and other forms of abuse.
-	api.staticRouter.POST("/login", api.userLoginHandler)
-	api.staticRouter.POST("/user", api.userHandlerPOST)
-	api.staticRouter.GET("/user", Validate(api.userHandlerGET))
-	api.staticRouter.PUT("/user", Validate(api.userHandlerPUT))
-	api.staticRouter.POST("/user/password", Validate(api.userChangePasswordHandler))
-	//api.staticRouter.POST("/password/reset/request", api.passwordResetRequestHandler)
-	//api.staticRouter.POST("/password/reset/verify", api.passwordResetCompleteHandler)
-	//api.staticRouter.POST("/password/reset/complete", api.passwordResetCompleteHandler)
+	api.staticRouter.GET("/user", validate(api.userHandlerGET))
```

Comment:
Documentation and implementation are diverging quite a bit here, I get that you want to save as much of the documentation you wrote already but honestly most of this stuff is easily copy pastable so I'd probably just get rid of as much as possible and keep accounts to a bare minimum of LoC.

Reply:
I'm talking about the documentation and implementation in general. The documentation lists some endpoints that are not available. You can ignore the comment I was just getting at "perhaps we should not invest any time yet in the documentation" but the point about Karol having a reference point is a good one.